extensions/typescript-language-features/src/languageFeatures/organizeImports.ts

```diff
@@ -61,24 +81,53 @@ class OrganizeImportsCommand implements Command {
 	}
 }
 
+class OrganizeImportsCommand extends BaseOrganizeImportsCommand {
+	public static readonly id = 'organizeImports';
+	public static minVersion = API.v280;
+	public static title = localize('organizeImportsAction.title', "Organize Imports");
```

Comment:
These keys are different from the ones in package.nls.json. Should I just access the ones declared in there instead? How can I do that?

Reply:
You can't access those strings. What you have here looks fine